pkg/apiserver/apiserver.go

```diff
@@ -30,9 +30,9 @@ type controllerInformer interface {
 	GetOperatorConfig() *config.Config
 	GetStatus() *spec.ControllerStatus
 	TeamClusterList() map[string][]spec.NamespacedName
-	ClusterStatus(team, cluster string) (*spec.ClusterStatus, error)
-	ClusterLogs(team, cluster string) ([]*spec.LogEntry, error)
-	ClusterHistory(team, cluster string) ([]*spec.Diff, error)
+	ClusterStatus(team, namespace, cluster string) (*spec.ClusterStatus, error)
```

Comment:
1. Since we now move to multiple namespaces it might make sense to add extra API endpoints for debugging purposes such as:
a. List namespaces operator actually listens to.
b. List databases per namespace
c. List namespaces per team

2. Please add/change 1-2 examples of the new API in [the debugging documentation](https://github.com/zalando-incubator/postgres-operator#debugging-the-operator-itself)